tests/fake_lab/tasks/heating.py

```diff
@@ -4,8 +4,9 @@
 from bson import ObjectId
 
 from alab_management import BaseTask
+from tests.fake_lab.devices.furnace import Furnace
+
```

Comment:
It should be this line that causes issues. Change it to 

`from .devices.furnace import Furnace` so that it will not run `add_device` twice